core/test/XiosFile_test.cpp

```diff
@@ -38,10 +38,14 @@ MPI_TEST_CASE("TestXiosFile", 2)
     config << "[model]" << std::endl;
     config << "start = 2023-03-17T17:11:00Z" << std::endl;
     config << "time_step = P0-0T01:30:00" << std::endl;
+    config << "[XiosInput]" << std::endl;
+    config << "period = P0-0T03:00:00" << std::endl;
+    config << "filename = xios_test_input" << std::endl;
```

Comment:
Note that any `filename` found in the `XiosInput` section will now get `setFileMode` with `'read'`. This will open the file provided, so the test needs to be set up such that the file exists and has the correct dimensions, fields, etc. This is why the dimensions, fields, etc. change in this test, too.